packages/atlas/src/components/_overlays/SendTransferDialogs/SendFundsDialog.tsx

```diff
@@ -25,12 +25,14 @@ import { hapiBnToTokenNumber, tokenNumberToHapiBn } from '@/joystream-lib/utils'
 import { useMemberAvatar } from '@/providers/assets/assets.hooks'
 import { useFee, useJoystream, useTokenPrice } from '@/providers/joystream/joystream.hooks'
 import { useTransaction } from '@/providers/transactions/transactions.hooks'
+import { formatJoystreamAddress, shortenAddress } from '@/utils/address'
 import { SentryLogger } from '@/utils/logs'
 import { formatNumber } from '@/utils/number'
 
 import { FormFieldsWrapper, PriceWrapper, StyledMaxButton, VerticallyCenteredDiv } from './SendTransferDialogs.styles'
 
-const ADDRESS_LENGTH = 49
+const POLKADOT_ADDRESS_LENGTH = 48
```

Comment:
Where does this value come from? I think some formats are shorter than this, my Kusama address uses 47 chars

Reply:
Do you think [this validator](https://github.com/Joystream/atlas/pull/3336/files#diff-f952c324408bcd44638e0d7bb9e3812efc268849074fc0af406df08d15e19f4aR13) is fine? I found this in [Polkadot docs](https://polkadot.js.org/docs/util-crypto/examples/validate-address/)